src/makeCalculator.js

```diff
@@ -4,7 +4,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
+      return result;
+    },
+
+    add: (num) => {
+      result += num;
+
+      return this;
+    },
+
+    subtract: (num) => {
+      result -= num;
+
+      return this;
+    },
+
+    multiply: (num) => {
+      result *= num;
+
+      return this;
+    },
+
+    divide: (num) => {
+      result /= num;
+
+      return this;
```

Comment:
Arrow functions do not have their own 'this' context, so 'this' inside the arrow functions for 'add', 'subtract', 'multiply', and 'divide' will not refer to the calculator object. Use regular function expressions to ensure the proper 'this' context is used.